core/scrollbar.js

```diff
@@ -599,7 +599,8 @@ Blockly.Scrollbar.prototype.setVisible = function(visible) {
  * @private
  */
 Blockly.Scrollbar.prototype.onMouseDownBar_ = function(e) {
-  this.onMouseUpHandle_();
+  Blockly.Touch.clearTouchIdentifier(); // This is really a click.
```

Comment:
Comments on the end of a line should have two spaces before them.  // Like this.